Header 	header
float32[] values
float32   min_angle
float32   max_angle
float32   step
uint8	  nsectors


